### PR TITLE
chore: audio context lazy

### PR DESCRIPTION
**Pull request by FrolYo**
After some discussion we came to decision that we don't need audio context pool, but instead we need just lazy audio context

### CODE REVIEW

File: src/dotnet/Users.UI.Blazor/Pages/UserInvitePage.razor [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
     private async Task OnNewInviteClick() {
         var invite = new Invite {
-            Remaining = 10,
+            Remaining = 50,
```
Comment (alexis-kochetov) [inline src/dotnet/Users.UI.Blazor/Pages/UserInvitePage.razor:30]: Please roll-back unrelated changes

File: src/nodejs/src/audio-context-lazy.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    private async create() {
+        const audioContext = new AudioContext({
+            latencyHint: 'interactive',
+            sampleRate: 48000,
+        });
+
+        console.log(`${LogScope}.initialize: audioContext.state=`, audioContext.state);
+        await audioContext.resume();
```
Comment (alexis-kochetov) [inline src/nodejs/src/audio-context-lazy.ts:32]: Are you sure you don't need anything like that 
```
    if (audioContext.state === 'suspended')
        await audioContext.resume();
```

File: src/nodejs/src/audio-context-lazy.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    constructor() {
+        this.addInitEventListeners();
+    }
+
+    public async get(): Promise<AudioContext> {
+        if (!this.audioContext) {
+            await this.whenInitialized;
+        }
+
+        return this.audioContext;
```
Comment (alexis-kochetov) [inline src/nodejs/src/audio-context-lazy.ts:16]: Probably we need to check state there and call resume if state is not expected

**Review by alexis-kochetov — CHANGES_REQUESTED** [review]

File: src/dotnet/Users.UI.Blazor/Pages/UserInvitePage.razor (hunk 1 above)
Comment (FrolYo) [reply src/dotnet/Users.UI.Blazor/Pages/UserInvitePage.razor]: ok...

File: src/nodejs/src/audio-context-lazy.ts (hunk 2 above)
Comment (FrolYo) [reply src/nodejs/src/audio-context-lazy.ts]: done

File: src/nodejs/src/audio-context-lazy.ts (hunk 3 above)
Comment (FrolYo) [reply src/nodejs/src/audio-context-lazy.ts:18]: done

**Review by alexis-kochetov — APPROVED** [review]
